src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -122,13 +116,69 @@ set_num_subsets(const int new_num_subsets)
   return this->num_subsets;
 }
 
+template <typename TargetT>
+void
+PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin<TargetT>::
+set_proj_matrix(const shared_ptr<ProjMatrixByBin>& arg)
+{
+    this->PM_sptr = arg;
+}
+
+template<typename TargetT>
+void
+PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin<TargetT>::
+set_proj_data_info(const ProjData& arg)
+{
+    this->proj_data_info_sptr = arg.get_proj_data_info_sptr()->create_shared_clone();
+    if(this->skip_lm_input_file)
+    {
+        std::cout << "Dummy LM file" << std::endl;
+        this->list_mode_data_sptr.reset(new ListModeData_dummy(
+                                            arg.get_exam_info_sptr(),
+                                            proj_data_info_sptr));
+        this->frame_defs = arg.get_exam_info_sptr()->get_time_frame_definitions();
+    }
+    max_ring_difference_num_to_process = this->proj_data_info_sptr->get_scanner_sptr()->
+            get_num_rings() - 1;
```

Comment:
This seems quite dangerous. As no time to do this properly, use `warning()` and add a `\todo` in the doxygen of this function.

Reply:
If you come up with a better suggestion on where to have this, I am happy to move it. 
I am not sure that we need it for the Summer school. But better have it and fix it later than not have it and forget that we need it. I will add the warning, though